[BUG] release publish `make build` on github ubuntu-latest error

**Describe the bug**
 release publish `make build` on github ubuntu-latest error
```
# pkg-config --cflags  -- devmapper
Package devmapper was not found in the pkg-config search path.
Perhaps you should add the directory containing `devmapper.pc'
to the PKG_CONFIG_PATH environment variable
No package 'devmapper' found
pkg-config: exit status 1
# github.com/containers/storage/drivers/btrfs
Error: ../../../go/pkg/mod/github.com/containers/storage@v1.43.0/drivers/btrfs/btrfs.go:9:10: fatal error: btrfs/ioctl.h: No such file or directory
    9 | #include <btrfs/ioctl.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:247: bin/kbcli.linux.amd64] Error 2
Error: Process completed with exit code 2.
```
reference: https://github.com/apecloud/kubeblocks/actions/runs/3928280923/jobs/6715930376

**To Reproduce**
Steps to reproduce the behavior:

**Expected behavior**
publish release  all success.

**Screenshots**
![image](https://user-images.githubusercontent.com/109708205/212826093-027adc6e-4048-49a4-b5ea-0044ce91ad1a.png)

--- a/.github/workflows/cicd-merge.yml
+++ b/.github/workflows/cicd-merge.yml
@@ -13,6 +13,12 @@ jobs:
     name: make test in minikube
     steps:
       - uses: actions/checkout@v3
+      - name: install lib
+        run: |
+          sudo apt-get install -y --no-install-recommends \
+            libbtrfs-dev \
+            libdevmapper-dev
+
       - name: setup Go
         uses: actions/setup-go@v3
         with:

--- a/.github/workflows/release-image.yml
+++ b/.github/workflows/release-image.yml
@@ -49,6 +49,8 @@ jobs:
           RELEASE_VERSION="latest"
           if [[ ! -z "${{ github.event.inputs.image_tag }}" ]]; then
             RELEASE_VERSION="${{ github.event.inputs.image_tag }}"
+          elif [[ "${{ env.RELEASE_VERSION }}" == "main" ]]; then
+            RELEASE_VERSION="latest"
           elif [[ ! -z "${{ env.RELEASE_VERSION }}" ]]; then
             RELEASE_VERSION=`bash ${{ github.workspace }}/.github/utils/utils.sh ${{ env.RELEASE_VERSION }} 1`
           fi

--- a/.github/workflows/release-publish.yml
+++ b/.github/workflows/release-publish.yml
@@ -36,6 +36,12 @@ jobs:
         os: [linux-amd64, linux-arm64, darwin-amd64, darwin-arm64, windows-amd64]
     steps:
       - uses: actions/checkout@v3
+      - name: install lib
+        run: |
+          sudo apt-get install -y --no-install-recommends \
+            libbtrfs-dev \
+            libdevmapper-dev
+
       - name: Setup Go
         uses: actions/setup-go@v3
         with:

